.github/workflows/visual-tests.yml

```diff
@@ -43,7 +43,7 @@ jobs:
           path: |
             packages/*/test/visual/base/screenshots/*/failed/*.png
   lumo:
-    name: Lumo
+    name: Lumo (JS files)
```

Comment:
Let's keep the "Lumo" name for now as it's used as a required check and the web components release build restores that check after publishing to npm. Otherwise we would have to mark it as not required 🤔

Reply:
Done.